### ISSUE TITLE
There's a cropperMinWidth but not a cropperMinHeight

### ISSUE THREAD

**Issue by jaco-terbraak** [Mawi137/ngx-image-cropper#130]
I noticed there's a cropperMinWidth but not a cropperMinHeight. Is this intentional or just forgotten? It would be nice to have both.

**Comment by Mawi137** [comment]
So far cropperMinWidth was the only one asked for. cropperMaxHeight will be a little more difficult since the cropper takes the maximum possible size for the given aspectRatio. Setting a maximum number of pixels means that it needs to check its size, adjust and center the cropper if needed.

**Comment by imbdb** [comment]
Till this feature is added to main module, You can add the [Stackblitz](https://stackblitz.com/edit/image-cropper) `image-cropper` folder in your project

add below line to `image-cropper.component.ts` after line 62

`    @Input() cropperMinHeight = 0;
`

and change the function `setCropperScaledMinSize` on line 287 to the below code
```
private setCropperScaledMinSize(): void {
        if (this.originalImage && (this.cropperMinWidth > 0 || this.cropperMinHeight)) {
            this.cropperScaledMinWidth = this.cropperMinWidth > 0
            ? Math.max(20, this.cropperMinWidth / this.originalImage.width * this.maxSize.width)
            : 20;
            this.cropperScaledMinHeight = this.maintainAspectRatio
                ? Math.max(20, this.cropperScaledMinWidth / this.aspectRatio)
                : this.cropperMinHeight > 0
                ? Math.max(20, this.cropperMinHeight / this.originalImage.height * this.maxSize.height)
                : 20;
        } else {
            this.cropperScaledMinWidth = 20;
            this.cropperScaledMinHeight = 20;
        }
    }
```

Hope this helps.

**Comment by Mawi137** [comment]
You know there's something called "Pull requests"? You can simply fork this repository, make these changes on your forked repo. When you're ready, you open a pull request, I can review the changes and if all is fine, I'll merge it into this repo.

Suppose I don't want to merge your changes, but you still want to use the changes you made, you can simply use your forked branch instead of the version code of npm in your package.json.

**Comment by imbdb** [comment]
I tried to fork and make the changes locally but was not able to build the module correctly.
I have raised an issue #162 for the same.
There is no information regarding how to build and how to contribute in the readme.

**Comment by C0nstan** [comment]
I would also appreciate this feature.

**Comment by Mawi137** [comment]
Just released version `1.3.9` with this feature!
